modules/ROOT/pages/single-sign-on-with-oidc.adoc

```diff
@@ -150,50 +151,66 @@ The expected format for Keys and certificates is PEM (with or without the commen
+ ** `auth-url` needs to be set with the URL of your OIDC provider Authorization endpoint. It can be prefixed with `+{auth-server-url}+` to only indicate the path of the end point or be a full absolute URL. This property is specific to Bonita OIDC module and is not supported/documented by Keycloak.
+ ** `token-url` needs to be set with the URL of your OIDC provider Token endpoint. It can be prefixed with `+{auth-server-url}+` to only indicate the path of the end point or be a full absolute URL. This property is specific to Bonita OIDC module and is not supported/documented by Keycloak.
+ ** `logout-url` needs to be set with the URL of your OIDC provider Logout endpoint. It can be prefixed with `+{auth-server-url}+` to only indicate the path of the end point or be a full absolute URL. This property is specific to Bonita OIDC module and is not supported/documented by Keycloak.
+ ** `token-issuer-uri` needs to be set with the URL that will be used as issuer in the JWT tokens (`iss` claim). It can be prefixed with `+{auth-server-url}+` to only indicate the path of the issuer URI or be a full absolute URI. For example using Google's OIDC provider, the issuer is `https://accounts.google.com`. This property is specific to Bonita OIDC module and is not supported/documented by Keycloak.
+ ** `user-info-url` needs to be set with the URL of your OIDC provider userInfo end point. It is only used when accessing a resource with a Bearer `Authorization` header with an access token that is opaque (not a JWT) and cannot be validated or used to retrieve the logged in user claims. It can be prefixed with `+{auth-server-url}+` to only indicate the path of the end point or be a full absolute URL. This property is specific to Bonita OIDC module and is not supported/documented by Keycloak.
+ ** `jwks-url` needs to be set with the URL of your OIDC provider JSON Web Key Set endpoint. It can be prefixed with `+{auth-server-url}+` to only indicate the path of the end point or be a full absolute URL. This property is specific to Bonita OIDC module and is not supported/documented by Keycloak.
+ ** `jwt-access-token` indicates if the access token is a JWT. If false, it will be considered by Bonita OIDC module as an opaque token and its validation and retrieval of the logged in user principal name will be delegated to the userInfo endpoint (see `user-info-url`). This property is specific to Bonita OIDC module and is not supported/documented by Keycloak.
+ ** `scopes` can be used to set the value of the `scope` parameter sent to the OIDC provider in the authentication request. The scope value `openid` is automatically added if it is no present, but you can add other scope values like `email` or `profile` depending on you OIDC provider configuration and the claims you need in the JWTs. This property is specific to Bonita OIDC module and is not supported/documented by Keycloak.
```

Comment:
probably add more information about the scopes,  it's not very clear for me what is the usage of this field in oidc steps authenication.

Reply:
done